ext/js/data/options-util.js

```diff
@@ -788,7 +788,7 @@ export class OptionsUtil {
                     {action: 'addNoteTermKanji',  key: 'KeyE',      modifiers: ['alt'],  scopes: ['popup', 'search'], enabled: true},
                     {action: 'addNoteTermKana',   key: 'KeyR',      modifiers: ['alt'],  scopes: ['popup', 'search'], enabled: true},
                     {action: 'playAudio',         key: 'KeyP',      modifiers: ['alt'],  scopes: ['popup', 'search'], enabled: true},
-                    {action: 'viewNote',          key: 'KeyV',      modifiers: ['alt'],  scopes: ['popup', 'search'], enabled: true},
+                    {action: 'viewNotes',         key: 'KeyV',      modifiers: ['alt'],  scopes: ['popup', 'search'], enabled: true},
```

Comment:
Don't modify previous updates. Instead, create a new update which modifies the settings to adjust `viewNote` to `viewNotes`

Reply:
Moved this change to `_updateVersion25`